packages/harpguru-core/src/components/scales-menu/utils/get-option-stack-props/get-option-stack-props.tsx

```diff
@@ -0,0 +1,97 @@
+import { useGlobal } from 'reactn'
+import React, { useCallback } from 'react'
+import {
+  getScale,
+  getScaleByDegreeIds,
+  getScaleIds,
+  ScaleCategory,
+} from 'harpparts'
+import type { DegreeIds } from 'harpparts'
+
+import type { OptionStackProps } from '../../../option-stack'
+import { OptionItem } from '../../../option-item'
+
+// TODO: It's a bit confusing having a getter util which
```

Comment:
I've tolerated it with the `getHarpCells` function for this long.. Perhaps it's not such a bad thing.

Reply:
If it's not confusing to have the .tsx suffix, it's still definitely not correct to have all these hooks in a `get...` file in a non-hooks folder.

Reply:
Done

Reply:
Not "Done". What I mean is I've come to peace with the need to have `.tsx` suffixes on getters rather than components sometimes.